src/index.tsx

```diff
@@ -16,4 +16,4 @@ root.render(
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// reportWebVitals();
```

Comment:
Remove?

Reply:
I did not think we need to start measuring performance yet.

Reply:
👍